main.js

```diff
@@ -0,0 +1,68 @@
+import Contextmenu from './src/javascript/contextmenu.js';
+import Contentview from './src/javascript/contentview.js';
+import Item from './src/javascript/item.js';
+
+const init = () => {
+  const itemTarget = document.getElementById('itemTarget');
+  const searchInput = document.getElementById('searchInput');
+  const contentview = document.getElementById('contentview');
+  const saveButton = document.getElementById('saveButton');
+  const contextMenu = new Contextmenu(itemTarget);
+  const contentView = new Contentview(contentview);
+  
+  let cache = getData();
+  if(!cache) cache = setData();
+  const cacheItem = JSON.parse(cache);
+  cacheItem.target = itemTarget;
+  cacheItem.menu = contextMenu;
+  cacheItem.view = contentView;
```

Comment:
p5; 요렇게 함수로 빼도 좋겠네요. 
```suggestion
const enrichCacheItem = (cacheItem, { target, menu, view} ) => ({
   ...cacheItem,
   target,
   menu,
   view
  })
```

Reply:
> p5; 요렇게 함수로 빼도 좋겠네요.

명확한 의미 파악을 위해서 제안해주신 코드처럼 함수화가 좋겠네요 👍